aten/src/THC/THCAllocator.cpp

```diff
@@ -4,21 +4,30 @@ THCIpcDeleter::~THCIpcDeleter() {
   int prev_device;
   THCudaCheck(cudaGetDevice(&prev_device));
   THCudaCheck(cudaSetDevice(device_));
-  THCudaCheck(cudaIpcCloseMemHandle(data_));
   THCudaCheck(cudaSetDevice(prev_device));
 }
 
 void deleteTHCIpcDeleter(void* ptr) {
   delete static_cast<THCIpcDeleter*>(ptr);
 }
 
-at::DataPtr THCIpcDeleter::makeDataPtr(void* data, int device) {
+// Refer to NB [CUDA IPC and the caching allocator] for more details
```

Comment:
nittiest of nits: the prevailing convention in the codebase is `Note [Blah blah]`, rather than NB ;)